### DISCUSSION TITLE
How to get the events/sessions for which data is available?

### DISCUSSION THREAD

**Discussion by TracingInsights** [theOehrly/Fast-F1#328] [General]
I'm wondering how f1-tempo.com implemented the functionality of populating the drop-down options with only the events where data is available.

**Comment by theOehrly** [comment]
FastF1's event schedule maybe? It also has a column that indicates whether timing and telemetry data are available (called 'F1ApiSupport').

**Comment by TracingInsights** [comment]
['F1ApiSupport' is set as true for future events as well]( https://github.com/theOehrly/f1schedule/blob/master/schedule_2023.json) but while exploring livetiming API I noticed we can get the finished events/sessions data from https://livetiming.formula1.com/static/2023/Index.json. I see there is a open issue about this #260

**Reply by theOehrly** [reply]
Ah, now I understand your question correctly. Yes, there is currently no way to easily tell exactly when after a session the data is available. I didn't think you were looking for such precise information. #260 might fix that by I haven't looked into it yet.
